core/arch/arm/kernel/generic_entry_a64.S

```diff
@@ -47,7 +47,8 @@
 		adr	x2, stack_tmp_stride
 		ldr	w1, [x2]
 		mul	x2, x0, x1
-		adr	x1, stack_tmp
+		adrp	x1, stack_tmp
+		add	x1, x1, :lo12:stack_tmp
```

Comment:
I don't think that is normally how it is done on a64, at least it isn't the code the C compiler generates. It's also likely slower on a64, although that probably doesn't matter as much here.